superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -84,11 +81,54 @@ const StyledContainer = styled.div`
       & > span {
         color: ${theme.colors.grayscale.light1};
         margin-left: ${theme.gridUnit * 4}px;
-        margin-top: ${theme.gridUnit * 6}px;
       }
     }
 
-    & .ant-tooltip-open {
+    & .viz-gallery {
+      border: 1px solid ${theme.colors.grayscale.light2};
+      border-radius: ${theme.gridUnit}px;
+      margin: ${theme.gridUnit}px 0px;
+      max-height: calc(100vh - ${ELEMENTS_EXCEPT_VIZ_GALLERY}px);
+      flex: 1;
+    }
+
+    & .footer {
+      flex: 1;
+      display: flex;
+      flex-direction: row;
+      justify-content: flex-end;
+      align-items: center;
+
+      & > span {
+        color: ${theme.colors.grayscale.light1};
+        margin-right: ${theme.gridUnit * 4}px;
+      }
+    }
+
+    &&&& .ant-steps-item-tail {
```

Comment:
Yowza... the ampersand is a good hack, but this is another level! :) Still better than `!important`

Not sure a way around it by looking, but maybe we can dig a little deeper later, and perhaps add a comment for now explaining why these are needed.

Reply:
I find this a bit confusing. What is this doing exactly?

Reply:
It's just a way to increase selector specificity. Generally, we use it when we don't have a way to declare a class for an internal element, and `!important` is not an option.

In this example, the original selector is:

```
.ant-steps-vertical > .ant-steps-item > .ant-steps-item-container > .ant-steps-item-tail {
    display: block;
}
```
With the &&&& we have a resulting selector with higher priority:

```
.css-130sj7k.css-130sj7k.css-130sj7k.css-130sj7k .ant-steps-item-tail {
    display: none;
}
```

@rusackas Should I spend more time trying to make it simpler? I don't know if we will find a more readable alternative. All the times that we did that, we ended up with a bunch of nested selectors, which for me are less readable.

Reply:
I'll add the following comment to help for now:
`/* extra ampersands used to boost selector specificity */`